benchmarks/sharded-bm/README.md

```diff
@@ -102,8 +102,16 @@ Debug UI works, just use the machine public IP.
 The first step is to create VMs to run the nodes.
 
 You should be able to use any valid forknet image with [forknet terraform recipes](https://docs.nearone.org/doc/mocknet-guide-7VnYUXjs2A).
-This [setup](https://github.com/Near-One/infra-ops/tree/tpsnet/provisioning/terraform/infra/network/mocknet/tpsnet) is an example infra suitable for the benchmark. You can do something similar and deploy it with:
+This [setup](https://github.com/Near-One/infra-ops/tree/tpsnet/provisioning/terraform/infra/network/mocknet/tpsnet) is an example infra suitable for the benchmark. You can make a copy then make the following modifications:
 
+1. in `main.tf`: update `unique_id` and `start_height`, then specify the number
```

Comment:
line 130 we had

`- The number of nodes deployed in terraform must match the number of nodes expected in the benchmark scenario.`

Look like the above line can be removed after this change